### DISCUSSION TITLE
Question about prop.err in RxODE simulation

### DISCUSSION THREAD

**Discussion by snailittle** [nlmixrdevelopment/RxODE#417] [General]
I am reading this page and have a question https://nlmixrdevelopment.github.io/RxODE/articles/RxODE-sim-var.html

For the example codes below, I am not sure how the prop.err is defined in the theta term.  Does RxODE automatically translate it into theta*EPS(1)?

mod <- RxODE({
  eff(0) = 1
  C2 = centr/V2*(1+**prop.err**);
  C3 = peri/V3;
  CL =  TCl*exp(eta.Cl) ## This is coded as a variable in the model
  d/dt(depot) =-KA*depot;
  d/dt(centr) = KA*depot - CL*C2 - Q*C2 + Q*C3;
  d/dt(peri)  =                    Q*C2 - Q*C3;
  d/dt(eff)  = Kin - Kout*(1-C2/(EC50+C2))*eff;
})
Adding the parameter estimates
The next step is to get the parameters into R so that you can start the simulation:
  
  theta <- c(KA=2.94E-01, TCl=1.86E+01, V2=4.02E+01,  # central 
             Q=1.05E+01, V3=2.97E+02,                # peripheral
             Kin=1, Kout=1, EC50=200, **prop.err=0**)      # effects

**Comment by mattfidler** [comment]
Hi @snailittle,

The parameter is not automatically converted to an `eps` style variable.  Rather, it is fixed to zero as if no residual variability exists in the simulation example.  

If you wish to have the simulation on the level of the individual, you would use a matrix to specify the `sigma` matrix;  That is:

```R
theta <- c(KA=2.94E-01, TCl=1.86E+01, V2=4.02E+01, # central
Q=1.05E+01, V3=2.97E+02, # peripheral
Kin=1, Kout=1, EC50=200) # effects

sigma <- lotri(prop.err ~  0.1)

rxSolve(mod, theta, sigma=sigma)
``` 
 
For clarity this may need to be dropped from the example.

In general, you can call parameters anything, you simply need to name the matrices to determine if it is a between subject variability or a unexplained variability parameter.  

Hopefully this clarifies things.
